### ISSUE TITLE
Info verification required

### ISSUE THREAD

**Issue by iemiku** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#6027]
Hi Team, 

one of the customers advised that article is incorrect: 

section: https://docs.microsoft.com/en-us/microsoftteams/devices/teams-ip-phones#currently-certified-ip-phones

info t oreview:    Audiocodes C448HD **(CAP mode only)** | C450HD_TEAMS_0.225 | March 2019

customer advised that he reached out to Audiocodes and they confirmed it's fully certified device for Teams, provided customer with new firmware (customer did not confirm if the same as in the table or not) and since update device is visible and fully manageable in Teams Admin Center. 

Customer requested to review the article and update accordingly.
-- | -- | --

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aaaef9e1-8f1c-ebf0-7856-4c007342a2c3
* Version Independent ID: 3eaf671f-375e-8787-9c66-3947c3606559
* Content: [Phones and Devices for Teams - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/devices/teams-ip-phones)
* Content Source: [Teams/devices/teams-ip-phones.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/devices/teams-ip-phones.md)
* Service: **msteams**
* GitHub Login: @dstrome
* Microsoft Alias: **dstrome**

**Comment by scanum** [comment]
@iemiku  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by lindspea** [comment]
@iemiku thank you for reaching out. Just for clarity, are you suggesting having a footnote added to the article regarding Audiocodes C448HD (CAP mode only) | C450HD_TEAMS_0.225 | March 2019  that it's a fully certified device for Teams, provided the customer has new firmware and since the update device is visible and fully manageable in Teams Admin Center.?

**Comment by iemiku** [comment]
Hi,

I found information provided by the customer in a support case that Audiocodes provided them with the latest firmware that after installation allowed this device to be fully managed via Teams Admin Center without being signed in in CAP mode.

Firmware version is unknown as customer did not want to waste more time in helping us update our info (as he was previously mislead by our documentation that’s why he reached out to Audiocodes). I believe best approach is for someone who is responsible for this info to reach to Audiocodes and verify official information from their side and what firmware is that etc.

I just tried to give the heads up that the document seems no longer accurate 😊

Good luck!

Best Regards,
İeva Mikučionytė
Support Engineer
Microsoft Teams and Skype for Business
+351 210603912
Working Hours: M-F 8:00am – 5:00pm PT
Manager: Paulo De Oliveira / Paulo.DeOliveira@microsoft.com<mailto:Paulo.DeOliveira@microsoft.com>
[cid:image002.jpg@01D6AB94.2BE3D7B0]
